src/components/dropdown-item/daikin-dropdown-item.ts

```diff
@@ -109,32 +109,34 @@ export class DaikinDropdownItem extends LitElement {
   private _focusableElement!: HTMLElement | null;
 
   private _handleClick(event: PointerEvent): void {
-    if (this.disabled) {
+    if (
+      this.disabled ||
+      (event.target as HTMLElement).tagName === "DAIKIN-CHECKBOX"
+    ) {
       return;
     }
 
-    this.dispatchEvent(new Event("select", event));
+    this._emitSelect();
```

Comment:
We should call `preventDefault` for events that we have handled.

```suggestion
    this._emitSelect();
    event.preventDefault();
```